package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -981,9 +986,11 @@ def run(self, **kwargs):
 
     def _selection_serial(self, universe, selection_str):
         selection = []
-        for ts in ProgressBar(universe.trajectory, verbose=True,
-                              total=universe.trajectory.n_frames):
+        pm = ProgressMeter(universe.trajectory.n_frames,
```

Comment:
ProgressMeter should not be here. It's been deprecated and removed everywhere as part of PR #2617 .

You seem to be reverting PR #2617!